sdk/textanalytics/Azure.AI.TextAnalytics/src/Generated/Models/ApiVersion.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.AI.TextAnalytics.Models
+{
+    /// <summary> Text Analytics API version (for example, v3.0). </summary>
+    internal readonly partial struct ApiVersion : IEquatable<ApiVersion>
```

Comment:
I could not add an autorest directive to prevent this struct from being created. For some reason it does not affect CodeGen. I tried:
``` yaml
directive:
- from: swagger-document
  where: $["parameters"]["ApiVersion"]
  transform: >
    $["x-ms-enum"] = {
      "modelAsString": false
    }
```

It's not a big deal, thought. I'm making the new type `internal`, and we have no code relevant code changes outside of generated code.

Reply:
@AlexanderSher:  Any thoughts on this?   IIRC, we had the same issue with Anomaly Detector right around the holidays.

Reply:
I'm marking this conversation as resolved so we can unblock this PR. Even though it would be more convenient to keep the code as it was, in the end it's just an internal change and it doesn't affect the behavior of the SDK.